velox/connectors/tpch/TpchConnector.cpp

```diff
@@ -31,21 +31,21 @@ RowVectorPtr getTpchData(
     memory::MemoryPool* pool) {
   switch (table) {
     case Table::TBL_PART:
-      return velox::tpch::genTpchPart(maxRows, offset, scaleFactor, pool);
+      return velox::tpch::genTpchPart(pool, maxRows, offset, scaleFactor);
```

Comment:
Unrelated change?

Reply:
It is related because the memory pool has been explicitly assigned with this change. Otherwise we have to use the deprecated API. So make this cleanup.